storage/transaction_results.go

```diff
@@ -5,9 +5,6 @@ import "github.com/onflow/flow-go/model/flow"
 // TransactionResults represents persistent storage for transaction result
 type TransactionResults interface {
 
-	// Store inserts the transaction result
-	Store(blockID flow.Identifier, transactionResult *flow.TransactionResult) error
```

Comment:
transaction results has to be stored along with other block related thing in an atomic transaction, therefore, the Store is removed, which is not currently being used any way